Library/Homebrew/cmd/update-report.rb

```diff
@@ -30,6 +30,8 @@ def update_report
         ohai "Homebrew has enabled anonymous aggregate user behaviour analytics"
         puts "Read the analytics documentation (and how to opt-out) here:"
         puts "  https://git.io/brew-analytics"
+        puts "Sleeping for 10 seconds. Press CTRL+C if you'd like to configure analytics"
```

Comment:
Yeah I know that, but internet people are silly and wouldn't be aware of that fact. I'm directly responding to [this](https://news.ycombinator.com/item?id=13035183) hn comment (ugh hn).

This message just further highlights it. If your package manager is "stalling" on output, you're probably more likely to see that - and they have one less reason to think homebrew is screwing them (lulz).

For the record, I'm 👍 💯 analytics.